src/language/en-US.json

```diff
@@ -101,6 +103,7 @@
     "address_alias_not_exist": "Address alias does not exist",
     "address_format_error": "Address format error",
     "address_invalid": "This address is invalid",
+    "address_list_cannot_be_empty": "At lease 1 none Opt In account must be selected from the list.",
```

Comment:
```suggestion
    "address_list_cannot_be_empty": "At lease 1 non-opted-in account must be selected from the list.",
```

Reply:
https://github.com/nemgrouplimited/symbol-desktop-wallet/pull/1288#issuecomment-796441710
OK, I got it.